code/mathematical-algorithms/gcd_and_lcm/gcd_and_lcm.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace gcd_and_lcm
```

Comment:
You should follow normal C# naming conventions, see [here](http://www.dofactory.com/reference/csharp-coding-standards).